molecule/README.md

```diff
@@ -0,0 +1,42 @@
+Molecule tests for the role.
+
+# Test Matrix
+
+Test options "flow down" thro table unless changed.
+
+test   | # partitions | # groups in partitions? | other
+---    | ---          | ---                     | ---
+test1  | 1            | N                       | 2x compute node, sequential names (default test)
+test1b | 1            | N                       | 1x compute node
+test1c | 1            | N                       | 2x compute nodes, nonsequential names
+test2  | 2            | N                       | 4x compute node, sequential names
+test3  | 1            | Y                       |
```

Comment:
nit: # groups implies a number, not yes/no

Reply:
Fixed by f49db32